templates/definition/charger/nrgkick-gen2.yaml

```diff
@@ -0,0 +1,21 @@
+template: nrgkick-gen2
+products:
+  - brand: NRGKick
+    description:
+      generic: Gen2-JSON
+requirements:
+  description:
+    de: NRGkick Gen2 Ladeeinheit via lokale REST JSON API
```

Comment:
is the JSON part anyhow relevant for the users?

Reply:
there exits also a MODBUS api for the nrgkick gen2, so to avoid confusion i think JSON is important here. In nrgkick gen2 dey call it also JSON or Modbus API

Reply:
Does it matter for the user? Modbus might be easier to use and a little less overhead. We‘ll add the one or other only.

Reply:
to be honest, for the user it wouldn't matter. the JSON API was just easier for me to test and start with.  I will rename everythin to be API agnostic. You'll have to decide if you want to accept the JSON integration or decline it. 

IMHO modbus will have lesser overhead, but i guess it doesn't matter much.